src/pretix/plugins/banktransfer/tasks.py

```diff
@@ -190,13 +175,32 @@ def _get_unknown_transactions(job: BankImportJob, data: list, event: Event=None,
     return transactions
 
 
+def get_prefix_event_map(organizer=None, event=None):
+    prefix_event_map = {}
+
+    if event:
+        events = [event]
+    elif organizer:
+        events = organizer.events.all()
+    else:
+        return {}
+
+    for event in events:
+        prefix_event_map[event.slug.upper()] = event
+        if 'banktransfer' in event.get_payment_providers():
+            custom_prefix = event.get_payment_providers()['banktransfer'] \
+                .settings.get('code_prefix', as_type=str)
```

Comment:
`event.get_payment_providers()` is comparatively expensive to call, so let's call it only once per event. We might even not call it at all and just access ``event.settings.get('payment_banktransfer_code_pretix')`` directly.